browser/js/orders/order.state.js

```diff
@@ -0,0 +1,55 @@
+app.config(function ($stateProvider) {
```

Comment:
I'd suggest splitting this file up. I get why it's together, but WRT `state` files, I would only include a parent state and its substates within a single file, it may be hard to find the other routes later on